src/Extension/Evaluator.cs

```diff
@@ -8,8 +9,11 @@ namespace Extension
 {
     public class Evaluator
     {
+        SortedDictionary<int, string> executionCriteria;
```

Comment:
Please suppress the XML comment warnings.

Reply:
See #28  😃